client/src/panels/panel_declarations/misc/key_concept_panels/key_concept_panels.js

```diff
@@ -26,23 +28,23 @@ const { TM } = create_text_maker_component([
   est_lang,
 ]);
 
-export const common_panel_config = {
+const common_panel_config = {
   is_static: true,
   footnotes: false,
   source: false,
   calculate: _.constant(true),
 };
 
-export const curried_render = ({
-  q_a_keys,
-  is_initially_expanded,
-  background_color,
-}) =>
-  function ({ calculations: { subject } }) {
-    let rendered_q_a_keys = _.compact([
-      ...q_a_keys,
-      subject.level === "crso" && "what_are_CR",
-    ]);
+export class FAQPanel extends React.Component {
+  render() {
+    const {
+      rendered_q_a_keys,
+      is_initially_expanded,
+      background_color,
+      subject,
+    } = this.props;
+
+    let subj = _.isUndefined(subject) ? Gov : subject;
```

Comment:
You can do this for default props:
```
class FAQPanel extends React.Component {
  ...
}
FAQPanel.defaultProps = {
  subject: Gov,
}
```

Reply:
While I think this is probably fine, adding props like `q_args`, `a_args` would give more flexibility
Alternatively, you can also change the structure of `q_a_keys =[ {key: "text_key", q_args: { Gov }, a_args: { Gov } }, {...} ]`

Reply:
Upto you what you want of those

Reply:
Oh, and I think this can move to `components`

Reply:
Should I storybook it too?

Reply:
sure